dashboard/app/views/levels/_admin.html.haml

```diff
@@ -42,6 +42,11 @@
                   - @level.properties['pages'].each do |page|
                     - page['levels'].each do |level_name|
                       %li= link_to level_name, level_path(Level.find_by_name(level_name))
+            - if @level.properties['project_template_level_name']
```

Comment:
can you not just do `- template_name = @level.properties['project_template_level_name']`?